iothub/service/src/DigitalTwin/Authentication/PnpConnectionStringCredential.cs

```diff
@@ -9,7 +9,7 @@ namespace Microsoft.Azure.Devices.Authentication
     /// <summary>
     /// Allows authentication to the API using a Shared Access Key
     /// </summary>
-    internal class SharedAccessKeyCredentials : IotServiceClientCredentials
+    internal class PnpConnectionStringCredential : IotServiceClientCredentials
```

Comment:
Adding the prefix "pnp" to our types could come across as a bit confusing; esp since the namespace and client name both have the term "DigitalTwin". 
I know that calling this digital twin is confusing given that ADT is another service of its own; what do you think about not adding any service-specific prefix to these credential types and instead deriving that info by looking at the namespace under which it lives? Or how about we call it "DigitalTwin"xxx?

Reply:
Without a prefix, the other credentials will clash with the base types. And IotHub prefix is added to the ones we use in all other clients. The only option is PnP or DigitalTwin. Given David's argument above, we can just add the DigitalTwin prefix.